test/fixtures/async-app/boot/boot.js

```diff
@@ -0,0 +1,10 @@
+var loopback = require('loopback');
+
+module.exports = function (app, done) {
+  process.nextTick(function () {
+    var MyModel = loopback.createModel('ASYNCMODEL', {});
```

Comment:
Nitpick: it's better to call `app.registry.createModel` instead of `loopback.createModel`. Not a big deal here, just FYI.